### ISSUE TITLE
Enhancement: Add file tags while saving attachment to Nextcloud

### ISSUE THREAD

**Issue by nelway42** [nextcloud/mail#604]
The ability to save email attachments into Nextcloud is a fantastic feature and is the key reason why this app is perfect in my deployment.   

We use the Nextcould Tags feature all the time to tag files based on various criteria.  To tag a new file that  I saved from an email, I have to go back to the Files view, navigate to the folder I just saved to, and add the file tags.  

It would be a great enhancement if file tags could be applied to the file at the time of saving, from within the mail app itself.

**Comment by Bugsbane** [comment]
Makes sense to me from a ux perspective. Not sure if this requirement is unique to the mail app though, of if there should be some Nextcloud-wide "save/add file to Nextcloud" dialogue that includes the option of adding tags. Is there anywhere else this would be used?

**Comment by nelway42** [comment]
Yes actually, now that you mention it I agree with you.  The desktop and mobile clients don't have a good solution for tagging either and I would consider that a good enhancement.
For the mail app specifically, I'm already within the web interface (where tagging is well supported), and it's the 'back and forth' between the mail app and the files app that causes the frustration.
